<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startDialog</class>
 <widget class="QDialog" name="startDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>800</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>740</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>521</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>In this game you will be learning about password security. You will choose a master passwrod and depending on it's strength you will get alloted lives. Be careful though, there are some nasty enemies out there that want to crack your password. So defend against them with password towers (patent pending). To get a tower all you need to do is buy it with your points. To start you have fifty points and can only buy a single tower. You can get more points for beating rounds and defeating enemies. To place a tower just select a tower from the right hand menu. Place the tower where you want on the map (Note: Towers will only be placed if you have enough points). If you change your mind or don't have enough points press the cancel button. After placing your tower, select a password for it. After you are satisfied with your towers press start to start the round. After the round or during the round if you want, place more towers. You can click on your tower to upgrade it's password. Upgrades cost points and improve the range and damage of the tower. After you complete each round you can upgrade your master password for free which gives you more lives. Have fun!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="towers">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Towers</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="passwordTower">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>640</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordTower_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordTower_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="towers_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enemies</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="passwordTower_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>530</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordTower_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordTower_6">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>530</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordTower_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>640</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>xxxxxxxxxxxxxx</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
